e2e/support/helpers/e2e-ui-elements-helpers.js

```diff
@@ -30,6 +30,10 @@ export function appBar() {
   return cy.get("#root header").first();
 }
 
+export function appContent() {
```

Comment:
Created this helper to avoid cypress errors caused by ambiguity between the collection name in the sidebar and the one in the table.